patches/chrome-android-java-src-org-chromium-chrome-browser-ChromeTabbedActivity.java.patch

```diff
@@ -11,3 +11,11 @@ index a168b085348c218c2951964beb37e0a25d754836..ad5cf672b9548a4f9921c4bc3521a0c8
      @IntDef({BackPressedResult.NOTHING_HAPPENED, BackPressedResult.HELP_URL_CLOSED,
              BackPressedResult.MINIMIZED_NO_TAB_CLOSED, BackPressedResult.MINIMIZED_TAB_CLOSED,
              BackPressedResult.TAB_CLOSED, BackPressedResult.TAB_IS_NULL,
+@@ -1161,6 +1161,7 @@ public class ChromeTabbedActivity
+                     CipherFactory.getInstance().restoreFromBundle(getSavedInstanceState());
+ 
+             boolean noRestoreState =
++                    super.isNoRestoreState() ||
```

Comment:
can't we just set this command line option?

Reply:
You were right, addressed in https://github.com/brave/brave-core/pull/3428/commits/acb75f104578bcbc1bba91e752f88c12cf5a54ba.
No patching is required now, thank you!